app/src/test/java/com/duckduckgo/app/systemsearch/SystemSearchViewModelTest.kt

```diff
@@ -528,15 +539,15 @@ class SystemSearchViewModelTest {
         val suggestion = AutoCompleteHistorySuggestion(phrase = "phrase", title = "title", url = "url", isAllowedInTopHits = false)
         val omnibarText = "foo"
 
-        val testObserver = TestObserver.create<String>()
-        testee.resultsPublishSubject.subscribe(testObserver)
+//        val testObserver = TestObserver.create<String>()
+//        testee.resultsStateFlow.subscribe(testObserver)
```

Comment:
You can delete the commented-out lines in this file

Reply:
Thanks! Removed :)